src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -18,12 +18,13 @@
     "require": {
         "php": ">=5.3.9",
         "symfony/security": "~2.7|~3.0.0",
-        "symfony/http-kernel": "~2.2|~3.0.0"
+        "symfony/http-kernel": "~2.2|~3.0.0",
+        "ircmaxell/password-compat": "dev-master"
```

Comment:
Also, is it really a hard requirement? It's only useful on older versions of PHP.